mteb/tasks/Reranking/multilingual/MIRACLReranking.py

```diff
@@ -65,7 +65,7 @@ class MIRACLReranking(MultilingualTask, AbsTaskReranking):
         modalities=["text"],
         eval_splits=[_EVAL_SPLIT],
         eval_langs=_LANGUAGES,
-        main_score="NDCG@10(MIRACL)",
+        main_score="ndcg_cut_10",
```

Comment:
Question: What happened to the MIRACL retrieval and rerankers?

Reply:
Yes, still there - MIRACL needs a more significant change to convert to the new format, so I wanted to make sure we were okay with this direction before I spent more time changing everything.

`NDCG@10(MIRACL)` will be replaced with `ndcg_cut_10`, AFAIK they are the same but the previous reranking setup didn't compute NDCG.

Reply:
We should probably test that to make sure.

Reply:
Absolutely! I plan to test all the reranking scores, want to make sure they're consistent.